config/DetectorParams.cfg

```diff
@@ -55,15 +55,15 @@ gen.add("markerBorderBits",                       int_t,    0,
 
 gen.add("maxErroneousBitsInBorderRate",           double_t, 0,
```

Comment:
these settings are biasing the detector toward type 1 errors which seems to be at odds with the concerns about false positives that I've heard various complaints about over the last year. Whats the rationale here?